vue/src/vueTextMask.js

```diff
@@ -0,0 +1,49 @@
+import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
+
+/*
+ * Vue Directive - Text Masking with Vanilla Text Mask
+ * https://github.com/text-mask/text-mask/tree/master/vanilla
+ *
+ * This directive binds input masking via Text Mask. Once the element is
+ * destroyed it will be unbinded. Pass an options object, same as you would
+ * normally, as a parameter on the element being bound.
+ */
+
+/**
+ * Initialize variable to store directive's `this` so that we can use its
+ * context in inputHandler called via event listener
+ */
+let instance
```

Comment:
I think code in this scope gets executed once per application, which means there's only one `instance` per application, which means you cannot use more than one Text Mask field. We should remove this.